store/ssmstore.go

```diff
@@ -114,6 +126,12 @@ func (s *SSMStore) Write(ctx context.Context, id SecretId, value string) error {
 		return err
 	}
 
+	if len(tags) > 0 {
+		if err := s.WriteTags(ctx, id, tags, false); err != nil {
```

Comment:
❓ (More of the WriteTags method) - It looks like the WriteTags also take in `deleteOtherTags`, curious what would happen for the following case:
1. If there already exist a tag for a secret, let's say it has tag`app = backstage` and another tag `requiredChamber = true`
2. I then now call write tag for this secret with the `app = backstage`  and with `deleteOtherTags` is set to true, it will remove the `required = chamber`, but would it error out if we are trying to add the tag that already exist?

Reply:
If I understand your scenario correctly, it would be fine, because you can use `chamber tag write` to replace the value of an existing tag, and it's fine if the new value is the same as the old value.

In the implementation of `WriteTags`, first all of the tags which are not being written are deleted; because the `app` tag is being written, it isn't deleted. Then, the `app` tag is updated using the SSM add-tags-to-resource API, which either adds or overwrites tags.

Let me know if I misunderstood your scenario!

Reply:
makes sense! thank you for walking me through it!